packages/react-components/react-tag-picker/library/src/components/TagPicker/useTagPicker.ts

```diff
@@ -126,7 +126,7 @@ const childrenToTriggerAndPopover = (children?: React.ReactNode) => {
     trigger = childrenArray[0];
     popover = childrenArray[1];
   } else if (childrenArray.length === 1) {
-    popover = childrenArray[0];
+    trigger = childrenArray[0];
```

Comment:
isn't this a breaking change? Previously you could use a TagPicker without the input, now if you only provide one child to the TagPicker, it must be the input

Reply:
You're right, let's discuss this more